### PR TITLE
Update Holesky docs

### PR DESCRIPTION
**Pull request by teddyknox**

### CODE REVIEW

**Review by wesfloyd — APPROVED** [review]
looks great - thanks for the updates

File: holesky/README.md [hunk 1]
```diff
@@ -1,182 +1,182 @@
...
+:::info
+Please ensure you have reviewed the [current Active Operator Set cap](https://docs.eigenlayer.xyz/operator-guides/avs-installation-and-registration/eigenda-operator-guide#eigenda-churn-approver) and ensure you have sufficient delegated restaked ETH TVL before proceeding.
+:::
 
-> NOTE: For any Docker based commands, if you have installed as root then you might have to append `sudo` in front of the command.
+#### Step 1: Install Prerequisites
 
-### Core operations
-* Register your operator to EigenLayer using [EigenLayer CLI](https://github.com/Layr-Labs/eigenlayer-cli/blob/master/README.md)
+- Complete the EigenLayer CLI installation and registration [here](https://docs.eigenlayer.xyz/operator-guides/operator-installation).
```
Comment (jianoaix) [inline holesky/README.md:9]: can we separate this EL registration step? There were operators who missed this step and then failed to opt-in EigenDA and got confused

File: holesky/README.md [hunk 2]
```diff
@@ -1,182 +1,182 @@
-## Installation of EigenDA using docker
+# Holesky
 
-### Prerequisites:
-* Docker installed
-* [EigenLayer CLI](https://github.com/Layr-Labs/eigenlayer-cli) installed
+:::info
+Please ensure you have reviewed the [current Active Operator Set cap](https://docs.eigenlayer.xyz/operator-guides/avs-installation-and-registration/eigenda-operator-guide#eigenda-churn-approver) and ensure you have sufficient delegated restaked ETH TVL before proceeding.
+:::
```
Comment (ian-shim) [inline holesky/README.md:5]: does this get formatted as intended?

**Review by ian-shim — APPROVED** [review]

File: holesky/README.md (hunk 2 above)
Comment (teddyknox) [reply holesky/README.md:5]: Nope, will updated.

File: holesky/README.md (hunk 1 above)
Comment (teddyknox) [reply holesky/README.md:9]: Yes I'll separate these two steps and place emphasis on the registration part.

File: holesky/README.md [hunk 3]
```diff
@@ -1,182 +1,182 @@
...
+- Have more than 1.1x current lowest-stake Operator in the active Operator set. Please see [EigenDA Churn Approver](https://docs.eigenlayer.xyz/operator-guides/avs-installation-and-registration/eigenda-operator-guide#eigenda-churn-approver) for more detail.
+- The operator to churn out has less than 10.01% of the total stake
+
+Execute the following command to opt-in to EigenDA AVS:
+This command also downloads the latest SRS points if they don't exist on the node. The file is approximately 8GB in size and the opt-in process can some time to complete depending on the network bandwidth.
+
+Note: EigenDA enables the following Quorum settings.
+Quorums Maps:
+
+- StakedEth Quorum:  `0`
```
[the remark this answers is not in the document]
Comment (teddyknox) [reply holesky/README.md:64]: Would welcome a follow-up PR, I'm just copying it over. Wasn't it you that wrote this doc most recently?

**Review by ian-shim — APPROVED** [review]

**Review by jianoaix — APPROVED** [review]

**Pull request merged**